README.md

```diff
@@ -127,27 +126,77 @@ const myStamp = compose(config, warnOnCollisions);
 ```
 
 
-### Stamp `options` Reserved Keys
+### Stamp Options
 
-The following are reserved keys for the stamp options object:
+It is recommended that stamp initializers only use one argument: The stamp `options` argument. There are no reserved properties and no special meaning. However, using multiple arguments for a stamp could create conflicts where multiple stamps expect the same argument to mean different things. Using named parameters, it's possible for stamp creator to resolve conflicts with `options` namespacing. For example, if you want to compose a database connection stamp with a message queue stamp:
 
 ```js
-{
-  instance // The object to be mutated by the stamp
-}
+const db = dbStamp({
+  host: 'localhost',
+  port: 3000,
+  onConnect() {
+    console.log('Database connection established.');
+  }
+});
+
+const queue = messageQueueStamp({
+  host: 'localhost',
+  port: 5000,
+  onComplete() {
+    console.log('Message queue connection established.');
+  }
+});
+```
+
+If you tried to compose these directly, they would conflict with eachother, but it's easy to namespace the options at compose time:
+
+```js
+const DbQueue = compose({
+  initializers: [({db, queue}, { instance }) => {
+    instance.db = dbStamp({
```

Comment:
No, they can't, and they shouldn't. It's a contrived example missing stamp details, but the point is to show that *different stamps* with conflicting options can be combined into a single stamp with namespaced options, and a message queue shouldn't be telling the user that a database connection was established. ;)